modules/oke/variables.tf

```diff
@@ -53,6 +53,10 @@ variable "use_node_pool_volume_encryption" {
 
 variable "node_pool_volume_kms_key_id" {}
 
+variable "node_pool_node_metadata" {}
```

Comment:
This should be a map right. Please ensure you test with multiple node pools

Reply:
added variable type in submodule.